lib/datadog/tracing/contrib/sidekiq/server_internal_tracer/heartbeat.rb

```diff
@@ -44,6 +44,11 @@ def heartbeat
                 super
               end
             end
+
+            # Sidekiq v7.0.1 changes the functionality of the Sidekiq::Launcher#heartbeat method
+            # and introduces the Sidekiq::Launcher#beat method, which does the same thing as the
+            # Sidekiq::Launcher#heartbeat method in versions of Sidekiq prior to v7.0.1
+            alias beat heartbeat
```

Comment:
I recommend keeping the main code closer to the latest library version, and add an alias for the "legacy" versions.
In this case, rename the `def heartbeat` method above to `beat`, and change the alias to `alias heartbeat beat`.